test/unit/css.js

```diff
@@ -1323,11 +1323,11 @@ QUnit.test( "Keep the last style if the new one isn't recognized by the browser
 	assert.equal( el.css( "position" ), "absolute", "The old style is kept when setting an unrecognized value" );
 	el = jQuery( "<div></div>" ).css( "position", "absolute" ).css( "position", " " );
 
-	// Support: Edge 14
+	// Support: Edge 14-15
```

Comment:
>> That works for them, but is just not acceptable when it can introduce ambiguity. 1.0.0 - 1.1.0-3.1.4 is a valid version range, but take away the spaces and its meaning is no longer clear.
>
> Looks ambiguous to me either way, I'm not sure what that means even.

I think @gibson042 refers to the fact that semver allows hyphens in version strings so `1.1.0-3.1.4` is a pre-release to `1.1.0`. When you have spaces surrounding the version range separator, it's clear whether a particular hyphen is part of a version string or a range separator.